src/coreclr/ilasm/writer.cpp

```diff
@@ -1289,18 +1454,59 @@ HRESULT Assembler::CreatePEFile(_In_ __nullterminated WCHAR *pwzOutputFilename)
 
     if (FAILED(hr=CreateTLSDirectory())) goto exit;
 
-    if (FAILED(hr=CreateDebugDirectory())) goto exit;
-
-    if (FAILED(hr=m_pCeeFileGen->SetOutputFileName(m_pCeeFile, pwzOutputFilename))) goto exit;
```

Comment:
Was it problematic in some way to set the file name here?

Reply:
Looks like a no-op change -- reverted.